sharing/process/index.md

```diff
@@ -0,0 +1,53 @@
+
+### SGID Index
+{: .text-left}
+
+It's quite simple to list [qualifying]({% link about/policy/sgid/index.md %}) data in the SGID Index. After you submit an issue in Porter using the "Add data to the SGID Index template, we will review your submission against the qualifications and reach out with any questions. Once we've worked together to resovle any questions or problems, we'll add your dataset to the Index and close the issue.
+
+### Open Data
+{: .text-left}
+
+Adding data to [opendata.gis.utah.gov](https://opendata.gis.utah.gov) requires a few additional steps to navigate the ArcGIS Online sharing mechanics. This time you'll use the "Add data to Open Data" issue template. After the issue is opened, you'll need to request access to the [Open Data Staging Group](https://utah.maps.arcgis.com/home/group.html?id=3d3bd0d238f24f45b2c4b84f1015a317) in ArcGIS Online. This is a staging group that allows us to review the dataset before sharing it through Open Data.
+
+Once we've verified the dataset meets the qualifications, you can share it with the appropriate Utah SGID ArcGIS Online group. All items in these groups are automatically added to Open Data. If this is your first time sharing a data with the particular group, you'll need to request access to the group.
+
+Datasets that languish in the staging group, or that are added to a Utah SGID group without approval, may be removed according to our [policies]({% link about/policy/open-data/index.md %}).
+
+### Open SGID and ArcGIS Online
+{: .text-left}
+
+The Open SGID is the latest evolution of the traditional "SGID as a single geospatial database" model. Maintaining data in the Open SGID requires the most work, both on the front end and for long-term maintenance. We encourage you to share your data via Open Data or the SGID Index whenever possible. However, we may pull some datasets into the Open SGID that otherwise belong elsewhere in order to maintain consistency and familiarity with previous versions of the SGID geodatabase. This is done at our discretion according to our [policies]({% link about/policy/open-sgid/index.md %}).
+
+After opening an issue with the "Add data to the Open SGID" template, you'll need to reach out and arrange the best way to transfer your data to us for the first review. We can download data from a cloud storage solution (Google Drive, Dropbox, etc), pull it in from an ArcGIS Online Map or Feature Service, set up an FTP share for you to upload it to, or just have you email us a zipped shapefile or GDB.
+
+Once we get your data, we will run [Sweeper](https://github.com/agrc/sweeper) on it to identify and fix any geometry errors that could cause problems with the database (Postgres complains about some things that Esri lets slide). We'll share the results of this with you as a courtesy; while you're under no obligation to change your source data, doing so will help streamline any future data updates.
```

Comment:
simplify - no need to throw anyone under a bus
```suggestion
Once we get your data, we will run [Sweeper](https://github.com/agrc/sweeper) on it to identify and fix common data errors. We'll share the results of this with you as a courtesy; while you're under no obligation to change your source data, doing so will help streamline any future data updates.
```

Reply:
thought: that makes sense. However, I also want to avoid the impression that we'll be going in and reviewing other people's data for accuracy, etc—"common data errors" leaves a lot of room for misinterpretation. I think people would also be puzzled (like we were) that data that works fine in their GDB is causing issues in Postgres. Maybe something like "fix empty and duplicate geometries." 

I was under the impression that sweeper also did geometry validation checks, but it looks like that was just stubbed out. Was that just part of the initial preparation for moving data to Open SGID?

Reply:
Fixing empty geometry and identifying duplicate rows and geometries is exactly what sweeper does so I'm good with that description. It also does the metadata checks as well right?

When brainstorming sweeper we wanted it to validate geometry but no one implemented it for some reasons.

What you are remembering is the difference of implementation for the isvalid method in sqlserver and postgresql. The cloudb has to run makevalid over every dataset after the data is loaded in pg.

Reply:
Looking at the docs it does do metadata, I just don't know which commands are run when we pull data in.

IIUC, any data getting pulled into Open SGID is fixed by makevalid at some point. Is it worth mentioning that here, or is that too in the weeds?

Reply:
Yah I'm not sure what @gregbunce does either. I assume it might be different than what is planned to be automated for the internal SGID but that isn't set up yet.

I think the makvalid is an implementation detail that invokes rule #1, "no one cares"